packages/cxl-lumo-styles/scss/gravity-forms.scss

```diff
@@ -18,6 +18,26 @@
   }
 }
 
+/**
+ * Built-in spinner tweaks.
+ *
+ * @todo: remove inline styles from customizer,
+ * Spinner hotfix @kyle 2020.07.16.
+ * @see https://app.clickup.com/t/232pth1
+ * @since 2022.02.15
+ */
```

Comment:
```suggestion
```

None of other styles has such extensive comments and I don't see the reason for this one being  special.

Reply:
@anoblet when you'll have time for a tweak?

Reply:
@pawelkmpt Updated